client/src/components/Home/AboutUs/Event/Event.js

```diff
@@ -3,56 +3,52 @@ import axios from "axios";
 import "bootstrap/dist/css/bootstrap.min.css";
 import planet from "./planet.svg";
 import mobilePlanet from "./mobile-planet.svg";
-import "./Notice.css";
+import "./Event.css";
 
-function Notice() {
-  const [notices, setNotice] = useState([
-    { title: "fetching notices", description: " . . ." },
+function Event() {
+  const [events, setEvent] = useState([
+    { title: "Recieving signals...", description: "Decrypting... Latest News..." },
```

Comment:
We can leave it the same. `{ title: "fetching events", description: " . . ." }`

Reply:
done!